### PR TITLE
WIP: single post timeline improvement

### PR DESCRIPTION
**Pull request by StCyr**
* Resolves: #756 
* Target version: master 

### Summary

This is work in progress

### CODE REVIEW

**Comment by jancborchardt** [comment]
@StCyr regarding the single post page, screenshots would be great for quick design review also from others. :)

File: src/mixins/accountMixins.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+ * This file provides global account related methods
+ */
+
+import serverData from './serverData'
+
+export default {
+	mixins: [
+		serverData
+	],
+	methods: {
```
Comment (juliushaertl) [inline src/mixins/accountMixins.js]: Please use a computed property to make sure it is properly updated if needed

File: src/mixins/serverData.js [hunk 2]
```diff
@@ -20,6 +20,22 @@
  *
  */
 
+/*
+ * This file provides global methods for using the serverData structure
+ * the serverData structure's data are initialy found in a DOM element
+ *  provided by the server.
+ *
+ * It may contain the following information:
```
Comment (juliushaertl) [inline src/mixins/serverData.js]: How about a jsdoc type definition for this? Then your IDE could give you hints when coding.

Comment (StCyr) [reply src/mixins/serverData.js]: ok. I'll do that

File: src/mixins/accountMixins.js (hunk 1 above)
Comment (StCyr) [reply src/mixins/accountMixins.js]: ok

**Comment by jancborchardt** [comment]
@StCyr what are the missing points here? :) Also there’s some conflicts to resolve

**Comment by icewind1991** [comment]
rebased.

Haven't verified how much of the linked issue is closed with this.

before:
image

after:
image

there is still room for improvement but imo it makes sense to merge this as is and do further improvements in new PRs so we don't end up with stale PRs again

**Pull request merged**